src/migrations/023.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE player
+  ADD COLUMN ip_address varchar(50) NOT NULL DEFAULT '_';
+
+CREATE TABLE blocked_ip (
+  id varchar(50) PRIMARY KEY NOT NULL,
+  address varchar(50) NOT NULL,
+  created_at TIMESTAMP WITHOUT TIME ZONE DEFAULT (NOW() AT TIME ZONE 'UTC')
+);
```

Comment:
POSIX standards demand that files end with newline characters.

`PRIMARY KEY` implies `NOT NULL`, though there's nothing wrong with being verbose.

Network addresses should be stored in [`inet`](https://www.postgresql.org/docs/9.5/datatype-net-types.html#DATATYPE-INET) types

What's the motivation for making this a separate table instead of a column on the Player table? It looks like one player can only be associated with one blocked_ip using this implementation anyway.